files/etc/skel/.xsession

```diff
@@ -1 +0,0 @@
-sugar
```

Comment:
- [ ] no explanation given for removing this, there is bound to be some good reason.

Reply:
I had earlier merged SL's code with my branch which merged a couple of old changes with my fork, so I had to revert that PR and cherry pick the changes, I guess this got deleted as a consequence. Wil add it again

Reply:
Why would you do that? Is it needed now after your earlier changes were merged?